sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/Hook/WebHook.cs

```diff
@@ -7,12 +7,15 @@
 namespace Azure.AI.MetricsAdvisor.Models
 {
     /// <summary>
+    /// A web hook is the entry point for all the information available from the Metrics Advisor service, and calls a user-provided api when an alert is triggered.
+    /// All alerts can be sent through a web hook.
```

Comment:
I'm not sure if there's much value in adding this last bit (the same can be said about Email hooks, right?).

Reply:
no, they are different. From the docs:

An email hook is the channel for anomaly alerts to be sent to email addresses specified in the Email to section. Two types of alert emails will be sent: Data feed not available alerts, and Incident reports which contain one or multiple anomalies.